models/identity-stitcher.js

```diff
@@ -21,12 +21,14 @@ const {
   zipObject
 } = require('lodash')
 const TestUserGuids = [
-  '49e1f2f9-55cc-6c10-58ff-b9b46ca79579' // test@test.com
+  '49e1f2f9-55cc-6c10-58ff-b9b46ca79579', // test@test.com
+  'fd8c0e67-d19e-49c5-94c2-ff737302d2bc' // bill.test@tester.com
```

Comment:
Should actually be `bill.test.tester@stage.com`

Reply:
Both emails are associated with GR entity, I picked one.